<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkWindow</class>
 <widget class="QMainWindow" name="LinkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="iplineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>192.168.20.56</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portlineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>5555</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="linkButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
